### PR TITLE
devx: add more resource links

### PR DESCRIPTION
**Pull request by jhchabran**
I added sentry and along the way, updated all slack links :)

### CODE REVIEW

File: content/departments/product-engineering/engineering/enablement/dev-experience/index.md [hunk 1]
```diff
@@ -38,9 +38,9 @@ The Dev Experience team, or DevX for short, is a team focused on improving the d
 
 ## Contact
 
-- **Discussions and questions**: #dev-experience channel and [developer experience GitHub discussions](https://github.com/sourcegraph/sourcegraph/discussions/categories/developer-experience)
+- **Discussions and questions**: [#dev-experience](https://sourcegraph.slack.com/archives/C01N83PS4TU) channel and [developer experience GitHub discussions](https://github.com/sourcegraph/sourcegraph/discussions/categories/developer-experience)
```
Comment (bobheadxi) [inline content/departments/product-engineering/engineering/enablement/dev-experience/index.md]: Slack links, e.g. `#dev-experience`, are automatically generated by the handbook: https://handbook.sourcegraph.com/departments/product-engineering/engineering/enablement/dev-experience#contact so no need to include links here

File: content/departments/product-engineering/engineering/enablement/dev-experience/index.md [hunk 2]
```diff
@@ -63,11 +66,11 @@ The DevX team currently has weekly sync meetings and biweekly retrospectives wit
 - [Team sync notes](https://docs.google.com/document/d/1Lm6GT-F4v9OTa5wxa1-AKLtNwlDkORbbeGjqVd9kWPg/edit)
 - [Retrospective notes](https://docs.google.com/document/d/1QR1It6KGccwWRpASH16J64QNkpHtngI4o2ttrGpVCwU/edit#)
 
-#### `sg` hack hour
+#### `sg` hacking hour
```
Comment (bobheadxi) [inline content/departments/product-engineering/engineering/enablement/dev-experience/index.md]: I think `hack hour` is more grammatically correct? 🤔 anecdotally a search for "hack hour" comes up with:

image

while "hacking hour" doesn't come up with any other usages

that said this seems a more rare term than I expected, but intuitively "hack hour" sounds more right to me

Comment (bobheadxi) [reply content/departments/product-engineering/engineering/enablement/dev-experience/index.md]: I'm struggling with other verbs though - `run hour` sounds just as weird as `running hour` 😅

**Review by bobheadxi — CHANGES_REQUESTED** [review]

File: content/departments/product-engineering/engineering/enablement/dev-experience/index.md (hunk 1 above)
Comment (jhchabran) [reply content/departments/product-engineering/engineering/enablement/dev-experience/index.md]: TIL! Thank you, I'll revert that :)

File: content/departments/product-engineering/engineering/enablement/dev-experience/index.md (hunk 2 above)
Comment (jhchabran) [reply content/departments/product-engineering/engineering/enablement/dev-experience/index.md]: It's just that's how it's named in the calendar and announced on Slack, so I went with it without thinking too much about it :

File: content/departments/product-engineering/engineering/enablement/dev-experience/index.md [hunk 3]
```diff
@@ -40,7 +40,7 @@ The Dev Experience team, or DevX for short, is a team focused on improving the d
 
 - **Discussions and questions**: #dev-experience channel and [developer experience GitHub discussions](https://github.com/sourcegraph/sourcegraph/discussions/categories/developer-experience)
 - **[Support](#support)**: `@dev-experience-support` in Slack
-- **GitHub**: [team/dev-experience](https://github.com/sourcegraph/sourcegraph/labels/team%2Fdev-experience) label and @sourcegraph/dev-experience team.
+- **GitHub**: [team/dev-experience](https://github.com/sourcegraph/sourcegraph/labels/team%2Fdev-experience) label and [@sourcegraph/dev-experience team](https://github.com/orgs/sourcegraph/teams/dev-experience).
```
Comment (bobheadxi) [inline content/departments/product-engineering/engineering/enablement/dev-experience/index.md]: GitHub team mentions are also generated, so link not needed here either :) https://handbook.sourcegraph.com/departments/product-engineering/engineering/enablement/dev-experience#contact

image

Comment (jhchabran) [reply content/departments/product-engineering/engineering/enablement/dev-experience/index.md]: I love this company.

**Review by bobheadxi — APPROVED** [review]

File: content/departments/product-engineering/engineering/enablement/dev-experience/index.md [hunk 4]
```diff
@@ -52,6 +52,9 @@ To collaborate, we use the following:
 - Internal team channel in #dev-experience-internal
 - [GitHub planning board](https://github.com/orgs/sourcegraph/projects/212)
 - Daily updates via [Geekbot](https://app.geekbot.com/dashboard/standup/90468/view/insights) to #dev-experience-updates
+- Tools and languages update feed in #dev-experience-notification
+- GitHub issues and pull-requests feed in #dx-github-feed
```
Comment (bobheadxi) [inline content/departments/product-engineering/engineering/enablement/dev-experience/index.md]: Actually are these for collaboration?

**Review by bobheadxi — CHANGES_REQUESTED** [review]

File: content/departments/product-engineering/engineering/enablement/dev-experience/index.md (hunk 4 above)
Comment (bobheadxi) [reply content/departments/product-engineering/engineering/enablement/dev-experience/index.md]: This section lists what we use to _collaborate_ as a team, for things we own/manage/use to do our day-to-day-work maybe a separate resources section or an addition to [Responsibilities](https://handbook.sourcegraph.com/departments/product-engineering/engineering/enablement/dev-experience#responsibilities) is more apt

(sorry for the many comments!)

Comment (bobheadxi) [reply content/departments/product-engineering/engineering/enablement/dev-experience/index.md]: Or maybe [Tech stack](https://handbook.sourcegraph.com/departments/product-engineering/engineering/enablement/dev-experience#tech-stack)?

Comment (jhchabran) [reply content/departments/product-engineering/engineering/enablement/dev-experience/index.md]: _Internal_ collaboration to be precise, but yes. I thought it'd be useful to document that. I just noticed for example that @taylorsperry wasn't on #dx-github-feed earlier today.

Comment (bobheadxi) [reply content/departments/product-engineering/engineering/enablement/dev-experience/index.md]: I don't think this is for internal team collaboration, which I think are more communication tools and avenues we have to collaborate with each other, which include things like creating docs, issues, and messaging - this definitely falls under Tech stack and/or Responsibilities IMO

Comment (jhchabran) [reply content/departments/product-engineering/engineering/enablement/dev-experience/index.md]: Makes sense. Will move stuff then :)

File: content/departments/product-engineering/engineering/enablement/dev-experience/index.md [hunk 5]
```diff
@@ -26,9 +26,12 @@ The Dev Experience team, or DevX for short, is a team focused on improving the d
   - [Developer experience support](#support)
   - [Developer experience newsletter](./newsletter.md)
 - Continuous integration
+
```
Comment (bobheadxi) [inline content/departments/product-engineering/engineering/enablement/dev-experience/index.md]:
```suggestion
```

File: content/departments/product-engineering/engineering/enablement/dev-experience/index.md [hunk 6]
```diff
@@ -26,9 +26,12 @@ The Dev Experience team, or DevX for short, is a team focused on improving the d
...
   - [`sourcegraph/sourcegraph` Buildkite pipelines](https://docs.sourcegraph.com/dev/background-information/continuous_integration#buildkite-pipelines)
   - [Continuous integration playbook](../../process/incidents/playbooks/ci.md)
   - [Buildkite agents](../../tools/infrastructure/index.md#buildkite-agents)
+  - [Sentry project](https://sentry.io/organizations/sourcegraph/issues/?project=6110304) for the CI pipeline.
+
```
Comment (bobheadxi) [inline content/departments/product-engineering/engineering/enablement/dev-experience/index.md]:
```suggestion
```

**Review by bobheadxi — APPROVED** [review]

**Comment by bobheadxi** [comment]
Thanks for being so responsive to comments! 😁

**Pull request merged**